src/Components/Results/Programs/ProgramCard.tsx

```diff
@@ -44,12 +58,16 @@ const ProgramCard = ({ program }: ProgramCardProps) => {
   return (
     <div className="result-program-container">
       {program.new && (
```

Comment:
I think that is might have been easier if you created a flags container that is positioned absolutely at the top left, that you can than put the flags in it. That way you don't need to calculate the offset.